File: lib/enums/event_channel_names.dart
/// This enum contains the differents event channel names used to receive
/// events from the Kotlin project
enum EventChannelNames {
  bluetoothChannel("xsens-dot-bluetooth-permission"),
  connectionChannel("xsens-dot-connection"),
  measuringChannel("xsens-dot-measuring"),
  measuringStatusChannel("xsens-dot-measuring-status"),
  recordingChannel("xsens-dot-recording"),
  scanChannel("xsens-dot-scan");

  const EventChannelNames(this.channelName);

  final String channelName;
}

File: android/app/src/main/kotlin/com/example/figure_skating_jumps/channels/enums/EventChannelNames.kt
package com.example.figure_skating_jumps.channels.enums

enum class EventChannelNames(val channelName: String) {
    BluetoothChannel("xsens-dot-bluetooth-permission"),
    ConnectionChannel("xsens-dot-connection"),
    FileExportChannel("xsens-dot-file-export"),
    MeasuringChannel("xsens-dot-measuring"),
    RecordingChannel("xsens-dot-recording"),
    ScanChannel("xsens-dot-scan")
}